packages/core/src/util/cell.ts

```diff
@@ -162,7 +162,8 @@ export const defaultMapStateToEnumCellProps = (
 ): StatePropsOfEnumCell => {
   const props: StatePropsOfCell = mapStateToCellProps(state, ownProps);
   const options: EnumOption[] =
-    ownProps.options !== undefined ? ownProps.options : props.schema.enum.map(enumToEnumOptionMapper) || [enumToEnumOptionMapper(props.schema.const)];
+    ownProps.options !== undefined ? ownProps.options :
+    props.schema.enum?.map(enumToEnumOptionMapper) || [enumToEnumOptionMapper(props.schema.const)];
```

Comment:
I would prefer `undefined` when there is no `const` instead of an "empty" option object. I think this should work:
```suggestion
    ownProps.options ||
    props.schema.enum?.map(enumToEnumOptionMapper) || 
    props.schema.const && [enumToEnumOptionMapper(props.schema.const)];
```